sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill.md

```diff
@@ -0,0 +1,83 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill
+applicable: SharePoint Online
+title: Start-SPOSiteOpticalCharacterRecognitionBackfill
+schema: 2.0.0
+author: FarreltinF
+ms.author: fanyi
+ms.reviewer:
+---
+
+# Start-SPOSiteOpticalCharacterRecognitionBackfill
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization
+
+Initiates a job to trigger the Optical Character Recognition (OCR) process for all files for the selected site.
+
+## SYNTAX
+
+```Powershell
+
+Start-SPOSiteBackfillOCR [-Site] <string> 
```

Comment:
this doesn't match the cmdlet name line 13.

also, be sure to use platyPS to generate the template for your documentation.

[https://dev.azure.com/onedrive/ODSP%20Wiki/_wiki/wikis/OSDP-Wiki.wiki/82618/SharePoint-Online-PowerShell-cmdlets?anchor=public-cmdlet-documentation](https://dev.azure.com/onedrive/ODSP%20Wiki/_wiki/wikis/OSDP-Wiki.wiki/82618/SharePoint-Online-PowerShell-cmdlets?anchor=public-cmdlet-documentation)